app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/coursedetail/uploads/view/CourseUploadsFragment.kt

```diff
@@ -74,6 +74,11 @@ class CourseUploadsFragment : Fragment(), CourseUploadsView {
             courseUrl?.let { courseUploadsPresenter.requestCourseUploads(it) }
         } else if (type == 2) {
             courseUploadList?.let { setData(it) }
+            if(courseUploadList==null){
```

Comment:
Instead of null check, we need to do empty check I think.

Reply:
Okay,I will change it